app/my-app/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.2"
```

Comment:
I don't think we can use this docker-compose directly - we're generating this in the kubernetes layer.

Reply:
Please remove this file before merging